api/LogicalPathApi.cpp

```diff
@@ -4,74 +4,84 @@
 
 #include <spdlog/spdlog.h>
 
-namespace io {
-namespace swagger {
-namespace server {
-namespace api {
-
-using namespace io::swagger::server::model;
-
-LogicalPathApi::LogicalPathApi(Pistache::Address addr)
-    : httpEndpoint(std::make_shared<Pistache::Http::Endpoint>(addr))
-{
-}
-
-void LogicalPathApi::init(size_t thr = 2) {
-    auto opts = Pistache::Http::Endpoint::options()
-        .threads(thr);
-    httpEndpoint->init(opts);
-    setupRoutes();
-}
-
-void LogicalPathApi::start() {
-    httpEndpoint->setHandler(router.handler());
-    httpEndpoint->serve();
-}
-
-void LogicalPathApi::shutdown() {
-    httpEndpoint->shutdown();
-}
-
-void LogicalPathApi::setupRoutes() {
-    using namespace Pistache::Rest;
-
-    Routes::Delete(router, irods::rest::base_url + "/logicalpath", Routes::bind(&LogicalPathApi::delete_handler, this));
-    Routes::Post(router, irods::rest::base_url + "/logicalpath/rename", Routes::bind(&LogicalPathApi::rename_handler, this));
-
-    // Default handler, called when a route is not found
-    router.addCustomHandler(Routes::bind(&LogicalPathApi::default_handler, this));
-}
-
-void LogicalPathApi::rename_handler(const Pistache::Rest::Request& request,
-                       Pistache::Http::ResponseWriter response)
-{
-    try {
-        this->rename_handler_impl(request, response);
-    }
-    catch (const std::runtime_error& e){
-        response.send(Pistache::Http::Code::Bad_Request, e.what());
-    }
-}
-
-void LogicalPathApi::delete_handler(const Pistache::Rest::Request& request,
-                       Pistache::Http::ResponseWriter response)
+namespace io
 {
-    try {
-        this->delete_handler_impl(request, response);
-    }
-    catch (const std::runtime_error& e){
-        response.send(Pistache::Http::Code::Bad_Request, e.what());
-    }
-}
-
-void LogicalPathApi::default_handler(const Pistache::Rest::Request& request,
-                               Pistache::Http::ResponseWriter response)
-{
-    response.send(Pistache::Http::Code::Not_Found, "The requested Logicalpath method does not exist");
-}
-
-}
-}
-}
-}
-
+    namespace swagger
+    {
+        namespace server
+        {
+            namespace api
```

Comment:
Hmm... forgot about these "generated" files. If we are going to modify it anyway, we might as well squash down the namespaces to `io::swagger::server::api`. Up to you

Reply:
I have no strong opinions, but that might just be because I've stared at it long enough that it appears normal. @korydraughn ?

Reply:
Yes, let's inline these like @alanking suggested.